__tests__/endpoints/articles.test.js

```diff
@@ -11,6 +11,71 @@ afterAll(async () => {
     await db.end();
 });
 
+describe("/api/articles", () => {
+    describe("GET", () => {
+        test("200: responds with an array of articles", () => {
+            return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then((response) => {
+                    expect(response.body.articles.length).toBe(data.articleData.length);
```

Comment:
Would prefer you to use hard code a number for the length for it to be less brittle